samples/word/20-lists/manage-formatting.yaml

```
@@ -0,0 +1,132 @@
+order: 2
+id: word-lists-manage-formatting
+name: List formatting
+description: Shows how to format a list.
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#insert-list").click(() => tryCatch(insertFormatList));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function insertFormatList() {
+          // Insert a list starting with the third paragraph then set numbering and bullet types of the list items.
+          await Word.run(async (context) => {
+            const paragraphs = context.document.body.paragraphs;
+            paragraphs.load("$none"); // No properties needed.
+
+            await context.sync();
+
+            // Start new list using the third paragraph.
```

When I first ran through the sample, it wasn't apparent to me the third paragraph was turned into a list. 
```suggestion
            // Start new list with the third paragraph as the first list item.
```